app.js

```diff
@@ -2,7 +2,7 @@ const express = require('express')
 const serverless = require('serverless-http')
 const axios = require('axios')
 
-const { lambda, getFuncName } = require('./modules/util')
+const { lambda, getFuncName } = require('./modules/lib/util')
```

Comment:
this was a fup by me introduced in https://github.com/EQWorks/legion/pull/79 (particularly the inaction in https://github.com/EQWorks/legion/pull/79/commits/0a6b6eae7ee1fd03dd76a9fca2342e9a22404e53)